SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -23,40 +23,60 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
+	*/
 	UPROPERTY(EditAnywhere, config, Category = "Entity Pool", meta = (ConfigRestartRequired = false, DisplayName = "Initial Entity ID Reservation Count"))
 	uint32 EntityPoolInitialReservationCount;
 
-	/** The minimum number of entity IDs available in the pool before a new batch is reserved */
-	UPROPERTY(EditAnywhere, config, Category = "Entity Pool", meta = (ConfigRestartRequired = false, DisplayName = "Pool Refresh Minimum Threshold"))
+	/** 
+	 * Specifies when the SpatialOS Runtime should reserve a new batch of entity IDs: the value is the number of un-used entity 
+	 * IDs left in the entity pool which triggers the SpatialOS Runtime to reserve new entity IDs
+	*/
+	UPROPERTY(EditAnywhere, config, Category = "Entity Pool", meta = (ConfigRestartRequired = false, DisplayName = "Pool Refresh Threshold"))
 	uint32 EntityPoolRefreshThreshold;
 
-	/** The number of entity IDs reserved when the minimum threshold is reached */
+	/** 
+	* Specifies the number of new entity IDs the SpatialOS Runtime reserves when `Pool refresh threshold` triggers a new batch.
+	*/
 	UPROPERTY(EditAnywhere, config, Category = "Entity Pool", meta = (ConfigRestartRequired = false, DisplayName = "Refresh Count"))
 	uint32 EntityPoolRefreshCount;
 
-	/** Time between heartbeat events sent from clients to notify the servers they are still connected. */
+	/** Specifies the amount of time, in seconds, between heartbeat events sent from a game client to notify the server-worker instances that it's connected. */
 	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (ConfigRestartRequired = false, DisplayName = "Heartbeat Interval (seconds)"))
 	float HeartbeatIntervalSeconds;
 
-	/** Time that should pass since the last heartbeat event received to decide a client has disconnected. */
+	/** 
+	* Specifies the maximum amount of time, in seconds, that the server-worker instances wait for a game client to send heartbeat events. 
+	* (If the timeout expires, the game client has disconnected.) 
+	*/
 	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (ConfigRestartRequired = false, DisplayName = "Heartbeat Timeout (seconds)"))
 	float HeartbeatTimeoutSeconds;
 
 	/**
-	 * Limit the number of actors which are replicated per tick to the number specified.
-	 * This acts as a hard limit to the number of actors per frame but nothing else. It's recommended to set this value to around 100~ (experimentation recommended).
-	 * If set to 0, SpatialOS will replicate every actor per frame (unbounded) and so large worlds will experience slowdown server-side and client-side.
-	 * Use `stat SpatialNet` in editor builds to find the number of calls to 'ReplicateActor' and use this to inform the rate limit setting.
+	 * Specifies the maximum number of Actors replicated per tick.
+	 * Default: `0` per tick  (no limit)
+	 * To avoid game client and server-worker instance slowdown, set the value to around `100`
```

Comment:
I'd remove this line. We're seeing internally that we want it much higher, or off entirely.